src/main/java/GameMenu.java

```diff
@@ -0,0 +1,42 @@
+public class GameMenu {
+
+    private InputFeed inputFeed;
+    private Display display;
+
+    public GameMenu(InputFeed inputFeed, Display display) {
+        this.inputFeed = inputFeed;
+        this.display = display;
+    }
+
+    public void show() {
+        display.gameOptions();
+    }
+
+    public Player createOpponent(String gameChoice) {
+        if (gameChoice.equals("1")) {
+            return new HumanPlayer("O");
+        } else {
+            return new ComputerPlayer(new RandomLocationGenerator(), "O");
+        }
+    }
+
+    public Game createGame() {
+        display.greet();
+        display.gameOptions();
+        Board board = new Board();
+        Game game = new Game(new HumanPlayer("X"), createOpponent(loopForValidInput(inputFeed.get())), board);
+        display.displayMarks(game.firstPlayer(), game.secondPlayer());
+        return game;
+    }
+
+    private String loopForValidInput(String input) {
+        while (!validInput(input)) {
+            display.invalidInput();
+            input = inputFeed.get();
+        }
+        return input;
+    }
+    private boolean validInput(String input) {
+        return (input.equals("1") || input.equals("2"));
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Position literals first in String comparisons](https://www.codacy.com/app/mollie-c-stephenson/TicTacToe/file/2534705885/issues/source?bid=3284885&fileBranchId=3284887#l40)